src/mon/FSCommands.cc

```diff
@@ -385,6 +384,17 @@ class SetHandler : public FileSystemCommandHandler
       return -EINVAL;
     }
 
+  bool confirm = false;
+  cmd_getval(cmdmap, "yes_i_really_mean_it", confirm);
+  if (var == "max_mds" && !confirm && mon->mdsmon()->has_health_warnings()) {
+    ss << "One or more file systems health warnings are present. Modifying "
```

Comment:
system

Reply:
It should be `file systems` IMO since "more" comes right before.

Reply:
"More" as I read it modifies "warnings", and "file system" is also used as an adjective to modify "warnings".

Reply:
You are right.

Reply:
I was reading it only partially due to a last review that asked me to `s/system/systems`. I should've read it full statement. Thanks!

Reply:
Fixed.

Reply:
@anthonyeleven BTW, it would be great if you would  use
```suggestion
    ss1 << "One or more file systems health warnings are present. Modifying "
```

Reply:
If you like.  I've found that accepting those via the web interface complicates things, ymmv.